onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -3251,10 +3526,11 @@ TEST(ReductionOpTest, OptimizeShapeForFastReduce_ReduceDimWithZero1b) {
   ASSERT_EQ(fast_axes, expected_fast_axes);
 }
 
+#ifndef USE_MIGRAPHX
```

Comment:
why does the test code need to be `ifdef'd` out? can the EP be excluded in the test instead?

Reply:
all reference to USE_MIGRAPHX removed.